SPECS-EXTENDED/squid/sources

```diff
@@ -0,0 +1,3 @@
+SHA512 (squid-5.7.tar.xz) = 624a39041a6ceda6c470dc0937616f1aa67200f3db02b4d74095d8d706ed31d6df5e0417dcacde45f6be40b617bee018849793d52c96a626aab32a2b182972aa
```

Comment:
I think we do not require this file.
We should nuke it and also any references to it.